RyanNoelk/OpenEats
# Feature/serving updates

The core update in this PR is adding support for dynamic servings. Issue #193. While I was in there I also removed react.createClass.

List of Changes:
- Factoring the recipe modules to remove react.createClass in support for ES6 classes.
- Adding round method to the Ingredients module. Max decimal points is 3.
- Adding dynamic servings to the recipe module.

To Test:
- Make sure the servings are updated properly.
- Make sure the page still loads properly.
